client_app/src/components/volunteer/Shelters.js

```diff
@@ -11,14 +11,15 @@ import { faCalendarDays, faArrowRight, faCircleXmark } from "@fortawesome/free-s
 import { useSpring, animated } from "@react-spring/web";
 import dayjs from 'dayjs';
 import CurrentSelection from "./CurrentSelection";
+import ShiftsData from "./ShiftsData";
 
 const Shelters = (props) => {
   let defaultRadius = "5";
   if (props.condensed) defaultRadius = "25";
   const [data, setData] = useState([]);
-  const [latitude, setLatitude] = useState(33.997103);
-  const [longitude, setLongitude] = useState(-118.4472731);
-  const [radius, setRadius] = useState(defaultRadius);
+  const [setLatitude] = useState(33.997103);
+  const [setLongitude] = useState(-118.4472731);
+  const [setRadius] = useState(defaultRadius);
```

Comment:
This change is causing problems, when I click on "show opportunities near me". You should undo this change.
```
Uncaught runtime errors:
×
ERROR
setLatitude is not a function
TypeError: setLatitude is not a function
    at setLocation (http://localhost:3000/static/js/bundle.js:7957:5)
ERROR
setLatitude is not a function
TypeError: setLatitude is not a function
    at setLocation (http://localhost:3000/static/js/bundle.js:7957:5)
```